Sources/StreamChatUI/Utils/UIImage+Extensions.swift

```diff
@@ -37,3 +38,41 @@ extension UIImage {
         return photoURL
     }
 }
+
+// MARK: - UIImage extension related to size
+
+extension UIImage {
```

Comment:
IMO we should not do this an extension of UIImage but instead expose some public functions/methods that you can use in your application.

use-case: I want to change user avatar, ideally I can copy paste our component and re-use our utilities

Reply:
Yes that was the plan, just like mentioned here: https://github.com/GetStream/stream-chat-swift/pull/1344#discussion_r685411028

According to @SagarSDagdu this a big refactor and big effort to change that.

Reply:
Agreed. I've introduced `StreamImageProcessor` and got rid of the extension.